src/runtime/virtcontainers/utils/retry/retry.go

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2020. Ant Group. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// copy from https://github.com/dragonflyoss/image-service/blob/master/contrib/nydus-snapshotter/pkg/utils/retry/retry.go
+package retry
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+)
+
+type RetryableFunc func() error
+
+var (
+	DefaultAttempts      = uint(10)
+	DefaultDelay         = 100 * time.Millisecond
+	DefaultMaxJitter     = 100 * time.Millisecond
```

Comment:
Couild you add `MS` to the end of these to make it clearer what units they are using?

Reply:
done